Tests/L0/SonarQubePreBuild/_suite.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference path="../../../definitions/mocha.d.ts"/>
+/// <reference path="../../../definitions/node.d.ts"/>
+/// <reference path="../../../definitions/Q.d.ts"/>
+
```

Comment:
This is the boiler plate required to run the tests. It is a typescript generating nodejs. I have copied it from an existing test and made minor changes to it.